docs/Project-Configuration/Project-Commands.md

```diff
@@ -768,6 +768,188 @@ Parameters:
+                    {
+                        "bucket": "genny-metrics",
+                        "path": "WaldoActor.QuxOperation",
+                        "prefix": "foobar_variant.2022_11_bar_patch_ghijk67890_23_01_01_06_24_48_0",
+                        "tags": [],
+                        "created_at": "2023-01-01T09:50:04.992453+00:00",
+                        "local_path": "/data/mci/12345abcdef/build/WorkloadOutput/reports/bar/CedarMetrics/WaldoActor.QuxOperation.ftdc",
+                        "permissions": "public-read",
+                        "convert_bson_to_ftdc": false
+                    }
+                ],
+                "metrics": [],
+                "sub_tests": []
+            },
+            {
+                "info": {
+                    "test_name": "FredActor.BazOperation",
+                    "trial": 0,
+                    "tags": [],
+                    "args": {}
+                },
+                "created_at": "2023-01-01T09:51:11.175000+00:00",
+                "completed_at": "2023-01-01T09:51:14.552000+00:00",
+                "artifacts": [
+                    {
+                        "bucket": "genny-metrics",
+                        "path": "FredActor.BazOperation",
+                        "prefix": "foobar_variant.2022_11_bar_patch_ghijk67890_23_01_01_06_24_48_0",
+                        "tags": [],
+                        "created_at": "2023-01-01T09:50:04.992453+00:00",
+                        "local_path": "/data/mci/12345abcdef/build/WorkloadOutput/reports/bar/CedarMetrics/FredActor.BazOperation.ftdc",
+                        "permissions": "public-read",
+                        "convert_bson_to_ftdc": false
+                    }
+                ],
+                "metrics": [],
+                "sub_tests": []
+            },
+        ]
+    }
+]
+```
+
+Each test result object holds data about a specific test and its
+subtests. It is represented by the
+[`Test` struct](https://pkg.go.dev/github.com/evergreen-ci/poplar#Test),
+and contains these fields:
+
+-   `info`: The test's `info` object, described below.
+-   `created_at`: The test's creation timestamp.
+-   `completed_at`: The test's completion timestamp.
+-   `artifacts`: The test's list of `artifact` objects, described below.
+-   `metrics`: The test's list of `metric` objects, described below.
+-   `sub_tests`: The test's list of subtest objects, which recursively
+    have the same format as the parent's test result object.
+
+**Note:** Although the `Test` struct includes the `_id` field, you
+should not populate it. It would be populated automatically by
+`perf.send`.
+
+Each `info` object holds metadata about the test configuration and
+execution. It is represented by the
+[`TestInfo` struct](https://pkg.go.dev/github.com/evergreen-ci/poplar#TestInfo),
+and contains these fields:
+
+-   `test_name`: The test's name, as a string.
+-   `trial`: An integer representing repeated execution (starts at 0,
```

Comment:
Flag as optional. I would also not used the word "execution" since that can be confused with a task execution, you can say something like `a repeated test run`. AFAIK, no one actually uses this but we can keep it around if we see value in it.

Reply:
Good idea, done!